### PR TITLE
fix: use same urn class in both read and write path in SCSI

### PR DESCRIPTION
**Pull request by jywadhwani**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by keremsahin1 — COMMENTED** [review]
You also need to change default entity type filter logic, I guess?

File: dao-impl/ebean-dao/src/main/java/com/linkedin/metadata/dao/EbeanLocalDAO.java [hunk 1]
```diff
@@ -371,8 +372,14 @@ private void updateUrnInLocalIndex(@Nonnull URN urn) {
     }
 
     final Map<String, Object> pathValueMap = _urnPathExtractor.extractPaths(urn);
+    final String entityType;
```
Comment (keremsahin1) [inline dao-impl/ebean-dao/src/main/java/com/linkedin/metadata/dao/EbeanLocalDAO.java:375]: Hmmm, you don't need to do this here (every time you index and urn into the index). Better to do this as part of the constructor?

**Review by keremsahin1 — APPROVED** [review]
Thanks for a much simpler fix

**Comment by jywadhwani** [comment]
Looks like build is failing. I will revert this change and try to submit it again.

**Pull request merged**